ajax/libs/ami.js/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "ami.js",
+  "npmName": "ami.js",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "ami.min.js",
+  "version": "0.0.11",
+  "description": "AMI Medical Imaging (AMI) JS ToolKit for THREEJS",
+  "keywords": [
+    "ami",
+    "ami.js",
+    "three.js",
+    "webgl",
+    "dicom",
+    "nifti",
+    "awesome",
+    "medical",
+    "imaging",
+    "xtk",
+    "nrrd",
+    "vtk",
+    "stl",
+    "trk"
+  ],
+  "homepage": "https://fnndsc.github.io/ami",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FNNDSC/ami"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name" : "Nicolas Rannou",
+    "url" : "https://github.com/NicolasRannou"
```

Comment:
oops, wrong indent here, no space before `:` please :) should pass the test this time